release-calendar/src/client/components/App.tsx

```diff
@@ -43,12 +46,17 @@ export class App extends React.Component<{}, AppState> {
 
   render(): JSX.Element {
     return (
-      <div>
+      <>
```

Comment:
?

Reply:
It's a React specific thing. https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
I see. Can you make it consistent? Either use `<>` or `<Fragment>` but not both.

Reply:
<> doesn't let you add keys which is needed for this line`<Fragment key={row.channel}>` so they are a little different. Should I write React.Fragment everywhere then?

Reply:
Yes please, let's keep it consistent.